bundles/org.openhab.binding.modbus.helioseasycontrols/README.md

```diff
@@ -0,0 +1,306 @@
+# Helios easyControls Binding
+
+Helios Heat-Recovery Ventilation devices use a Modbus protocol to communicate with different sensors, switches, etc. Some devices come with an integrated web interface (easyControls) as well as a Modbus TCP/IP Gateway. See https://www.easycontrols.net/de/service/downloads/send/4-software/16-modbus-dokumentation-f%C3%BCr-kwl-easycontrols-ger%C3%A4te for the corresponding specification.
+
+## Supported Things
+
+| Thing                           | Description                                                |
+|---------------------------------|------------------------------------------------------------|
+| helios-ventilation-easycontrols | Helios Heat-Recovery Ventilation devices with easyControls |
+
+## Configuration
+You first need to set up a Modbus bridge according to the Modbus documentation. Things in this extension will use the selected bridge to connect to the device. The configuration of a Helios Ventilation device via a `.things` file would look like the following code sample. It's required to provide the device's IP address, port and unit ID (port and unit ID cannot be changed and always have to be 502 and 180; in fact the values provided here are ignored by the binding and the fixed values 502 and 180 are used):
+```
+Bridge modbus:tcp:modbus-gateway "Modbus TCP/IP Gateway" [ host="x.x.x.x", port=502, id=180, enableDiscovery=true ] {
+    Thing helios-ventilation-easycontrols kwl "KWL"
+}
+```
+
+## Channels
+The following channels are supported:
+
+| Channel                          | Channel Group   | Description                                                      | Item Type                | Unit | Values                                                                                               |
+| -------------------------------- | --------------- | ---------------------------------------------------------------- | ------------------------ | ---- | ---------------------------------------------------------------------------------------------------- |
+| articleDescription               | general         | The KWL's article description                                    | String                   |      |                                                                                                      |
+| refNo                            | general         | The KWL's reference number                                       | String                   |      |                                                                                                      |
+| macAddress                       | general         | The KWL's MAC Address                                            | String                   |      |                                                                                                      |

```

Comment:
Are these rather properties than Channels?

Reply:
`articleDescription` and `refNo` are actually writeable, `macAddress` isn't. In any case these are also variables of the device provided via the same protocol as any other value.
What would be the best approach if these values were properties?

Reply:
Making these writeable sounds like some kind of configuration (which is not the intention of OH).

You can update properties with `updateProperty()`.